Assets/Scripts/Bee/NectarGlob.cs

```diff
@@ -23,6 +24,7 @@ void Update()
             {
                 onUse();
                 Destroy(gameObject);
+                nectarBar.GetComponent<NectarUI>().addHealth(0.1f);
```

Comment:
doesn't this script already GetComponent<NectarUI>() earlier in the Update method? What is the difference between getting it on the nectarBar vs just GetComponent<NectarUI>()?